analysis/topeft_run2/analysis_processor.py

```diff
@@ -117,7 +117,7 @@ def __init__(self, samples, wc_names_lst=[], hist_lst=None, ecut_threshold=None,
                 if hist_to_include not in self._accumulator.keys():
                     raise Exception(f"Error: Cannot specify hist \"{hist_to_include}\", it is not defined in the processor.")
             self._hist_lst = hist_lst # Which hists to fill
-
+        self._hist_lst = ["njets","lj0pt","ptz", "ptz_wtau"]
```

Comment:
Please remove. It's better to have the command line options and make these the default for `ana` in the run script.

Reply:
Agreed